### ISSUE TITLE
`BrowserWindow` needs to be focused for `webContents.sendInputEvent()` to work

### ISSUE THREAD

**Issue by josephrocca** [electron/electron#8299]
* Electron version: (don't know how to get this, `electron -v` doesn't work for me. it's a recent version though - downloaded a couple of weeks ago)
* Operating system: Ubuntu 16.04

### Expected behavior

`sendInputEvent` should work whether the window is focused or not.

### Actual behavior

`sendInputEvent` doesn't work unless window is focused.

### How to reproduce

```javascript
// (open window1 and window2)
window1.focus(); // remove this and the command below doesn't work
window1.webContents.sendInputEvent({
  type: "char",//"keyDown",
  keyCode: "b",//unicodeValue,
});
```

I tried searching past issues for this and I was surprised it hasn't been mentioned (maybe it's documented behaviour that I missed?). This doesn't affect me too much, but it took a little while to debug so I figured I'd post for the sake of others.

Cheers

**Comment by rajatsingla28** [comment]
Hi @josephrocca i am beginner here so could you confirm if the behaviour needs to be documented in web-contents.md?

**Comment by rajatsingla28** [comment]
Please review this PR #9615 and suggest the required changes.

**Comment by kevinsawicki** [comment]
This was fixed by #9615
